firmware/src/widgets/parqetWidget.cpp

```diff
@@ -340,75 +341,54 @@ void ParqetWidget::updatePortfolioChart()
 void ParqetWidget::clearScreen(int8_t displayIndex, int32_t background)
 {
     m_manager.selectScreen(displayIndex);
-    TFT_eSPI &display = m_manager.getDisplay();
-    display.fillScreen(background);
+    m_manager.fillScreen(background);
 }
 
 void ParqetWidget::displayClock(int8_t displayIndex, uint32_t background, uint32_t color, String extra, uint32_t extraColor)
 {
-    Serial.printf("displayClock at screen %d\n", displayIndex);
+    // Serial.printf("displayClock at screen %d\n", displayIndex);
```

Comment:
You have disabled multiple of these prints,
I am unsure about whether or not they need to be kept, but commented out is not the way to go.